src/adv.py

```diff
@@ -49,3 +99,22 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+print(
+    f"\n Welcome to your first adventure, {player.name}! \n Use 'n' to go North \n Use 's' to go South \n Use 'w' to go West \n Use 'e' to go East \n Press 'q' to quit game")
+
+while not inits[0] == 'q':
+    if current_room == room['outside']:
```

Comment:
What is the purpose of this? Wouldn't you want to print out the current room in every room?